page-components/links.md

```diff
@@ -20,19 +20,21 @@ Links lead users to a different page or further information. In contrast, <a hre
 </div>
 
 <div class="content-50 content-last">
-  <h5 class="repo-list-header">Repository</h5>
+  <h5 class="repo-list-header">Code Source</h5>
   <ul class="repo-list">
     <li>
       <span class="cf-icon cf-icon-github"></span>
     </li>
     <li>
-      <a href="https://github.com/cfpb/cf-typography"><h4>cf-typography</h4></a>
-      <p>Links in the Capital Framework</p>
+      <a href="https://github.com/cfpb/capital-framework/blob/canary/src/cf-typography/src/atoms/links.less">
```

Comment:
Like with the main Typography page, most of what this page discusses is set in cf-core. Change here?